scripts/circleci_usage_data.py

```diff
@@ -0,0 +1,267 @@
+#!/usr/bin/env python
+# Copyright 2014 Scalyr Inc.
```

Comment:
2020?

Reply:
I just copied the header from other file, but yeah, we should update all the files to say 2014-2020 (can be done in a separate PR :))